db/migrate/20210927092807_migrate_stripe_account_owner_to_stripe_sca.rb

```diff
@@ -0,0 +1,17 @@
+class MigrateStripeAccountOwnerToStripeSca < ActiveRecord::Migration[6.1]
+  def up
+    replace_preferences_key("/spree/gateway/strip_connect", "/spree/gateway/stripe_sca")
```

Comment:
You didn't test this, did you?
```suggestion
    replace_preferences_key("/spree/gateway/stripe_connect", "/spree/gateway/stripe_sca")
```